src/skimage/filters/_median.py

```
@@ -16,39 +16,40 @@ def median(
-        with the same number of dimension than ``image``.
-        If None, ``footprint`` will be a N-D array with 3 elements for each
-        dimension (e.g., vector, square, cube, etc.)
-    out : ndarray, (same dtype as image), optional
+        If ``behavior='rank'``, `footprint` is a 2-D array of 1's and 0's.
+        If ``behavior='ndimage'``, `footprint` is a N-D array of 1's and 0's
+        with the same number of dimensions as `image`.
+        If None, `footprint` will be a N-D array with 3 elements for each
+        dimension (e.g., vector, square, cube, etc.).
+    out : ndarray, same dtype as input `image`, optional
         If None, a new array is allocated.
     mode : {'reflect', 'constant', 'nearest', 'mirror','‘wrap'}, optional
```

Ruff (0.14.11)

Docstring contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)